src/test/java/subway/line/LineAcceptanceTest.java

```diff
@@ -0,0 +1,136 @@
+package subway.line;
+
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.annotation.DirtiesContext;
+import subway.line.dto.LineResponse;
+import subway.util.Extractor;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@DirtiesContext(classMode = DirtiesContext.ClassMode.AFTER_EACH_TEST_METHOD)
+@DisplayName("지하철노선 관련 기능")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+class LineAcceptanceTest {
+
+    private static Map<String, String> 신분당선;
+    private static Map<String, String> 분당선;
+
+    private static final String LINE_URL = "/line";
+
+    @BeforeEach
+    void init() {
+        신분당선 = getParams("신분당선", "bg-red");
+        분당선 = getParams("분당선", "bg-yellow");
+    }
+
+    /**
+     * When 지하철 노선을 생성하면
+     * Then 지하철 노선 목록 조회 시 생성한 노선을 찾을 수 있다
+     * */
+    @DisplayName("지하철 노선 생성")
+    @Test
+    void createLine() {
+
+        //given
+        //when
+        saveLine(신분당선);
+        //then
+        List<String> lines = Extractor.get(LINE_URL).jsonPath().getList("name", String.class);
+        assertThat(lines).containsAnyOf(신분당선.get("name"));
+    }
+
+    /**
+     * Given 2개의 지하철 노선을 생성하고
+     * When 지하철 노선 목록을 조회하면
+     * Then 지하철 노선 목록 조회 시 2개의 노선을 조회할 수 있다.
+     * */
+    @DisplayName("지하철노선 목록을 조회한다.")
+    @Test
+    void searchLines() {
+
+        //given
+        saveLine(신분당선);
+        saveLine(분당선);
+        //when
+        List<String> lines = Extractor.get(LINE_URL).jsonPath().getList("name", String.class);
+        //then
+        assertThat(lines).hasSize(2);
```

Comment:
containsExactly 메서드를 활용해서 추출된 노선명까지 함께 검증해도 좋을 것 같아요

```suggestion
        //then
        assertThat(lines).containsExactly("신분당선", "분당선");
```